# HG changeset patch
# User Rik <rik@octave.org>
# Date 1357174750 28800
#      Wed Jan 02 16:59:10 2013 -0800
# Node ID 11d5a7276b6dfd0ec95a2852d39a086b5305478c
# Parent  7d300b85ee255f31fe8ba059ec922f02266a711a
configure.ac: Remove some no longer accurate comments.

* configure.ac: Remove some no longer accurate comments.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2007,31 +2007,21 @@ double tgamma (double);
 AC_LANG_POP(C++)
 
 ## Look in <cmath> for the IEEE functions isnan, isinf, isfinite that we need.
 
 OCTAVE_CHECK_FUNC_CMATH(isnan)
 OCTAVE_CHECK_FUNC_CMATH(isinf)
 OCTAVE_CHECK_FUNC_CMATH(isfinite)
 
-dnl Would like to get rid of this crap, and just have
-dnl
-dnl   AC_CHECK_FUNCS([finite isnan isinf])
-dnl
-dnl instead, but that used to fail on some systems...
-dnl
-dnl Also just using AC_CHECK_FUNCS doesn't seem to work to find isinf
-dnl and isnan on Linux systems, so we use AC_CHECK_FUNC, and if that
-dnl fails, we try again by including math.h and invoking the function
-dnl with an argument. 
-
-### I am told that Inf and NaN don't work on m68k HP sytems.
+## Check for Inf and NaN functions
 
 case $canonical_host_type in
   m68k-hp-hpux*)
+    ## I am told that Inf and NaN don't work on m68k HP sytems.
   ;;
   *)
     AC_CHECK_FUNCS([finite isnan isinf signbit])
     AC_CHECK_FUNCS([_finite _isnan])
     AC_CHECK_DECLS([signbit], , , [#include <math.h>])
   ;;
 esac
 
